custom/mapContainers.libsonnet

```diff
@@ -37,16 +37,41 @@ local patch = {
   mapContainersWithName(names, f):: fn.mapContainersWithName(names, f),
 };
 
+local cronFn = fn + {
```

Comment:
Not 100% sure, but doesn't `jsonnetfmt` remove the `+`  here? Could you confirm using `jsonnetfmt` or `tk fmt`?

Reply:
TIL about `tk fmt`...